pipeline_dp/pipeline_backend.py

```diff
@@ -659,11 +659,28 @@ def singleton(self):
 class LocalBackend(PipelineBackend):
     """Local Pipeline adapter."""
 
+    def __init__(self, output_reiterable: bool = True):
+        """
+        Args:
+          output_reiterable: if true, output of all methods are reiterable,
+           i.e. it's possible to run iteration any number of time. It is
+           guaranteed that the iteration output is the same on each iteration.
+           Setting it to False, make things faster (~16%), and require ~2 times
+           less memory (no need to cache each output) but can lead to
+           errors.
```

Comment:
I think a little bit misleading, it cannot lead to errors if implemented correctly. Better "but it is more error-prone from coding perspective because you might try to iterate it more than once and get empty output".